app/code/Magento/Cms/view/adminhtml/web/js/folder-tree.js

```diff
@@ -101,8 +104,45 @@ define([
                             tree.jstree('select_node', folderEl);
                         });
                     }
+                },
+
+                /**
+                 * Get currentPath decode it returns new path array
+                 */
+                _parseCurrentPath = function () {
+                    var paths = [];
+
+                    $.each(encodedPath, function (i, val) {
+                        var isLastElement = i === encodedPath.length - 1;
+
+                        if (isLastElement) {
+                            paths[i] = window.MediabrowserUtility.pathId.replace(',,', '--');
+                        } else {
+                            paths[i] = Base64.encode(val).replace('==', '--');
```

Comment:
I think the replace here isn't correct and may cause other problems, please follow below the way PHP is handling the encode, I believe we have to perform the same behavior on JS.

file: app/code/Magento/Cms/Helper/Wysiwyg/Images.php
`    /**
     * Encode string to valid HTML id element, based on base64 encoding
     *
     * @param string $string
     * @return string
     */
    public function idEncode($string)
    {
        return strtr(base64_encode($string), '+/=', ':_-');
    }
`